src/lib/lmc-maps.ts

```diff
@@ -74,7 +77,21 @@ class LmcMaps {
 
         this.setControls();
 
-        this.hasMarker && createMarker(this.coords).addTo(this.map);
+        this.bounds = new mapboxgl.LngLatBounds();
+
+        this.coords.forEach((coord: mapboxgl.LngLatLike): void => {
+            if (this.hasMarker) {
```

Comment:
Proposal:
Remove hasMarker option. Use center and coords. If coords are defined, always render markers.

Reply:
hasMarker option preserved due to better configurability. Center point is set by center option even if there are multiple coordinates. If center options is set, fitBounds() is not used. Otherwise is fitBounds() used even if there is only one coordinate.